views/spots/form.njk

```diff
@@ -57,6 +57,16 @@
         </label>
     </div>
 
+    <label for="payload">
+        JSON payload
+        {% if errors.payload %}
+        <span class="error">Payload must be valid JSON</span>
+        <textarea id="payload" type="text" name="payload" placeholder="JSON payload" aria-invalid="true">{{ spot.payload }}</textarea>
+        {% else %}
+        <textarea id="payload" type="text" name="payload" placeholder="JSON payload">{{ spot.payload }}</textarea>
+        {% endif %}
+    </label>
+
```

Comment:
The current version is OK for me but we will have to validate it is clear for the end user. They might not know what a "JSON" is, neither what a "payload" is.

I would create a drop down menu called "Display advanced fields (mostly for developers)" at the end of the form containing this JSON payload field (and potentially others in the future) to clarify the interface.